SETUP/lint_charsuites.php

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env php
+<?php
+$relPath="../pinc/";
+include_once($relPath."CharSuites.inc");
+
+foreach(CharSuites::get_all() as $charsuite)
+{
+    echo "Validating charsuite $charsuite->name...\n";
+    if(!$charsuite->are_codepoints_normalized())
+    {
+        echo "    ERROR: codepoints are not normalized\n";
```

Comment:
It would be useful to print out what the character was.

Reply:
That's true. I struggled with how to best do that given the framework that I have right now but decided to punt on that for the time being.

Reply:
7 hours later I figured out how I wanted to do it and just did it.